.github/workflows/test-state-manager.yml

```
@@ -42,26 +46,23 @@ jobs:
+          flags: unit-tests
           name: state-manager-coverage-report
-          fail_ci_if_error: true
-
-      - name: Upload test results
-        uses: actions/upload-artifact@v4
-        if: always()
-        with:
-          name: state-manager-test-results
-          path: state-manager/pytest-report.xml
-          retention-days: 30
+          fail_ci_if_error: true
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)